src/Settings/components/reminders/ReminderOptions/utils/index.js

```diff
@@ -0,0 +1,54 @@
+import { updateReminders, constant, trackReminderEvent, STRINGS } from "@common";
+import { setReminderBanis } from "@common/actions";
+
+const setDefaultReminders = async (baniListData, dispatch, isReminders, reminderSound) => {
+  const baniList = baniListData;
+  const defaultReminders = () => {
+    return [
+      {
```

Comment:
Can we remove this redundant code using a loop? Basically we are creating objects with bani ids 0, 1, 19 and 21 at their specific times. Can we use a loop to generate this object?